### PR TITLE
New Mario & Luigi Superstar Saga songs (I messed up last pull)

### PR DESCRIPTION
**Pull request by ARand0-mGuy**
I made some mistakes but I fixed them so everything should be Okie Dokie now

### CODE REVIEW

File: data/Music/Mario & Luigi Series/Mario & Luigi Superstar Saga/Come On!-ARG.meta [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Mario & Luigi Superstar Saga - Come On!-ARG
```
Comment (PurpleHato) [inline data/Music/Mario & Luigi Series/Mario & Luigi Superstar Saga/Come On!-ARG.meta:1]: For consistency, would be cool to rename the title like this:
Mario & Luigi Superstar Saga - Come On! V2

**Review by PurpleHato — CHANGES_REQUESTED** [review]

File: data/Music/Mario & Luigi Series/Mario & Luigi Superstar Saga/Final Cackletta Fight.meta [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Mario & Luigi Superstar Saga - Final Cackletta Fight
```
Comment (PurpleHato) [inline data/Music/Mario & Luigi Series/Mario & Luigi Superstar Saga/Final Cackletta Fight.meta:1]: Mario & Luigi: Superstar Saga - Final Cackletta Fight

File: data/Music/Mario & Luigi Series/Mario & Luigi Superstar Saga/Popple Battle.meta [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Mario & Luigi Superstar Saga - Popple Battle
```
Comment (PurpleHato) [inline data/Music/Mario & Luigi Series/Mario & Luigi Superstar Saga/Popple Battle.meta:1]: Mario & Luigi: Superstar Saga - Popple Battle

**Review by PurpleHato — CHANGES_REQUESTED** [review]
I just noticed you also didn't put a : after the Mario & Luigi (still for being consistant)

**Comment by DaruniasJoy** [comment]
jesus this is messy xD add your tracks to the spreadsheet please

**Pull request merged**